src/disco/shred/fd_shred_tile.c

```diff
@@ -1039,6 +1039,17 @@ after_frag( fd_shred_ctx_t *    ctx,
         fec = fd_store_insert( ctx->store, ctx->round_robin_id, (fd_hash_t *)fd_type_pun( &ctx->out_merkle_roots[fset_k] ) );
       } FD_STORE_SHARED_LOCK_END;
 
+      if( FD_UNLIKELY( !fec ) ) {
+        /* fec can be null for several reasons, but the most likely case
```

Comment:
are there actually other reasons? i'd just explicitly state store_insert returns null if it's already been inserted